### PR TITLE
Develop autentication features change logs

### PR DESCRIPTION
**Pull request by mmed-c8y**

### CODE REVIEW

File: content/change-logs/platform-services/cumulocity-10-18-590-0-support-for-certificate-revocation-list.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: CRL support in device certificate authentication
+product_area: Platform services
+change_type:
+  - value: change-QHu1GdukP
+    label: Feature
+component:
+  - value: q3kclF6pO
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-18-590-0-support-for-certificate-revocation-list.md:9]: value is missing the "component-" prefix

File: content/change-logs/platform-services/cumulocity-10-20-6-0-the-administrator-cannot-change-passwords-for-other-users.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Administrators cannot change passwords for other users
+product_area: Platform services
+change_type:
+  - value: change-QHu1GdukP
+    label: Feature
+component:
+  - value: q3kclF6pO
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-6-0-the-administrator-cannot-change-passwords-for-other-users.md:9]: see above

File: content/change-logs/platform-services/cumulocity-10-18-590-0-support-for-certificate-revocation-list.md (hunk 1 above)
Comment (mmed-softwareag) [reply content/change-logs/platform-services/cumulocity-10-18-590-0-support-for-certificate-revocation-list.md:9]: hmm the same file was merge into y2024 and this file was created by netlify-cms

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-18-590-0-support-for-certificate-revocation-list.md (hunk 1 above)
Comment (mmed-softwareag) [reply content/change-logs/platform-services/cumulocity-10-18-590-0-support-for-certificate-revocation-list.md:9]: explained

File: content/change-logs/platform-services/cumulocity-10-20-6-0-the-administrator-cannot-change-passwords-for-other-users.md (hunk 2 above)
Comment (mmed-softwareag) [reply content/change-logs/platform-services/cumulocity-10-20-6-0-the-administrator-cannot-change-passwords-for-other-users.md:9]: explained

**Review by kalavathig1 — APPROVED** [review]

**Pull request merged**